<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  
  
  <link rel="shortcut icon" href="../img/favicon.ico">
  <title>Class: Human-gutMIGSBacteria - MIxS</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700" />

  <link rel="stylesheet" href="../css/theme.css" />
  <link rel="stylesheet" href="../css/theme_extra.css" />
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" />
  
  <script>
    // Current page data
    var mkdocs_page_name = "Class: Human-gutMIGSBacteria";
    var mkdocs_page_input_path = "Human-gutMIGSBacteria.md";
    var mkdocs_page_url = null;
  </script>
  
  <script src="../js/jquery-2.1.1.min.js" defer></script>
  <script src="../js/modernizr-2.8.3.min.js" defer></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
  <script>hljs.initHighlightingOnLoad();</script> 
  
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> MIxS</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="..">Index</a>
                    </li>
                </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">MIxS</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
    
    <li>Class: Human-gutMIGSBacteria</li>
    <li class="wy-breadcrumbs-aside">
      
    </li>
  </ul>
  
  <hr/>
</div>
          <div role="main">
            <div class="section">
              
                <h1 id="class-human-gutmigsbacteria">Class: Human-gutMIGSBacteria</h1>
<p>Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-gut</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/Human-gutMIGSBacteria">mixs.vocab:Human-gutMIGSBacteria</a></p>
<p><img alt="img" src="http://yuml.me/diagram/nofunky;dir:TB/class/[QuantityValue],[Human-gutMIGSBacteria&#124;submitted_to_insdc:string;investigation_type:investigation_type_enum;sample_name:string;project_name:string;experimental_factor:string%20%3F;env_package:env_package_enum%20%3F;subspecf_gen_lin:string%20%3F;num_replicons:string;extrachrom_elements:string%20%3F;estimated_size:string%20%3F;ref_biomaterial:string;source_mat_id:string%20%3F;pathogenicity:string%20%3F;biotic_relationship:biotic_relationship_enum%20%3F;specific_host:string%20%3F;host_spec_range:string%20%3F;health_disease_stat:health_disease_stat_enum%20%3F;trophic_level:trophic_level_enum%20%3F;encoded_traits:string%20%3F;rel_to_oxygen:rel_to_oxygen_enum%20%3F;isol_growth_condt:string;sample_collect_device:string%20%3F;sample_collect_method:string%20%3F;samp_mat_process:string%20%3F;nucl_acid_ext:string%20%3F;nucl_acid_amp:string%20%3F;lib_size:string%20%3F;lib_reads_seqd:string%20%3F;lib_layout:lib_layout_enum%20%3F;lib_vector:string%20%3F;lib_screen:string%20%3F;adapters:string%20%3F;seq_meth:seq_meth_enum;tax_ident:tax_ident_enum%20%3F;assembly_qual:assembly_qual_enum;assembly_name:string%20%3F;assembly_software:string;annot:string%20%3F;number_contig:string;feat_pred:string%20%3F;ref_db:string%20%3F;sim_search_meth:string%20%3F;tax_class:string%20%3F;compl_score:compl_score_enum%20%3F;compl_software:string%20%3F;url:string%20%3F;sop:string%20%3F;lat_lon(i):string%20%3F;depth(i):string%20%3F;geo_loc_name(i):string%20%3F;collection_date(i):string%20%3F;env_broad_scale(i):string%20%3F;env_local_scale(i):string%20%3F;env_medium(i):string%20%3F;gastrointest_disord(i):string%20%3F;liver_disord(i):string%20%3F;special_diet(i):special_diet_enum%20%3F;host_subject_id(i):string%20%3F;host_sex(i):host_sex_enum%20%3F;host_disease_stat(i):string%20%3F;ihmc_medication_code(i):string%20%3F;chem_administration(i):string%20%3F;host_body_site(i):string%20%3F;host_body_product(i):string%20%3F;host_diet(i):string%20%3F;host_last_meal(i):string%20%3F;host_family_relationship(i):string%20%3F;host_genotype(i):string%20%3F;host_phenotype(i):string%20%3F;ihmc_ethnicity(i):string%20%3F;host_occupation(i):string%20%3F;medic_hist_perform(i):string%20%3F;perturbation(i):string%20%3F;oxy_stat_samp(i):oxy_stat_samp_enum%20%3F;organism_count(i):organism_count_enum%20%3F;samp_store_dur(i):string%20%3F;host_symbiont(i):string%20%3F;samp_store_loc(i):string%20%3F;misc_param(i):string%20%3F]uses%20-.-&gt;[MIGSBacteria],[Human-gut]^-[Human-gutMIGSBacteria],[Human-gut],[MIGSBacteria]" /></p>
<h2 id="parents">Parents</h2>
<ul>
<li>is_a: <a href="../Human-gut/">Human-gut</a> - human-gut</li>
</ul>
<h2 id="uses-mixins">Uses Mixins</h2>
<ul>
<li>mixin: <a href="../MIGSBacteria/">MIGSBacteria</a> - Minimal Information about a Genome Sequence: cultured bacteria/archaea</li>
</ul>
<h2 id="attributes">Attributes</h2>
<h3 id="inherited-from-human-gut">Inherited from human-gut:</h3>
<ul>
<li><a href="../alt/">alt</a>  <sub>OPT</sub><ul>
<li>Description: Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</li>
<li>range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../chem_administration/">chem_administration</a>  <sub>OPT</sub><ul>
<li>Description: List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: agar [CHEBI:2509];2018-05-11T20:00Z None</li>
</ul>
</li>
<li><a href="../collection_date/">collection_date</a>  <sub>OPT</sub><ul>
<li>Description: The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: 2018-05-11T10:00:00+01:00 None</li>
</ul>
</li>
<li><a href="../depth/">depth</a>  <sub>OPT</sub><ul>
<li>Description: The vertical distance below local surface, e.g. For sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../elev/">elev</a>  <sub>OPT</sub><ul>
<li>Description: Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit</li>
<li>range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../env_broad_scale/">env_broad_scale</a>  <sub>OPT</sub><ul>
<li>Description: In this field, report which major environmental system your sample or specimen came from. The systems identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. were you in the desert or a rainforest?). We recommend using subclasses of ENVO’s biome class: http://purl.obolibrary.org/obo/ENVO_00000428. Format (one term): termLabel [termID], Format (multiple terms): termLabel [termID]|termLabel [termID]|termLabel [termID]. Example: Annotating a water sample from the photic zone in middle of the Atlantic Ocean, consider: oceanic epipelagic zone biome [ENVO:01000033]. Example: Annotating a sample from the Amazon rainforest consider: tropical moist broadleaf forest biome [ENVO:01000228]. If needed, request new terms on the ENVO tracker, identified here: http://www.obofoundry.org/ontology/envo.html</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: forest biome [ENVO:01000174] None</li>
</ul>
</li>
<li><a href="../env_local_scale/">env_local_scale</a>  <sub>OPT</sub><ul>
<li>Description: In this field, report the entity or entities which are in your sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. Please use terms that are present in ENVO and which are of smaller spatial grain than your entry for env_broad_scale. Format (one term): termLabel [termID]; Format (multiple terms): termLabel [termID]|termLabel [termID]|termLabel [termID]. Example: Annotating a pooled sample taken from various vegetation layers in a forest consider: canopy [ENVO:00000047]|herb and fern layer [ENVO:01000337]|litter layer [ENVO:01000338]|understory [01000335]|shrub layer [ENVO:01000336]. If needed, request new terms on the ENVO tracker, identified here: http://www.obofoundry.org/ontology/envo.html</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: litter layer [ENVO:01000338] None</li>
</ul>
</li>
<li><a href="../env_medium/">env_medium</a>  <sub>OPT</sub><ul>
<li>Description: In this field, report which environmental material or materials (pipe separated) immediately surrounded your sample or specimen prior to sampling, using one or more subclasses of ENVO’s environmental material class: http://purl.obolibrary.org/obo/ENVO_00010483. Format (one term): termLabel [termID]; Format (multiple terms): termLabel [termID]|termLabel [termID]|termLabel [termID]. Example: Annotating a fish swimming in the upper 100 m of the Atlantic Ocean, consider: ocean water [ENVO:00002151]. Example: Annotating a duck on a pond consider: pond water [ENVO:00002228]|air ENVO_00002005. If needed, request new terms on the ENVO tracker, identified here: http://www.obofoundry.org/ontology/envo.html</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: soil [ENVO:00001998] None</li>
</ul>
</li>
<li><a href="../gastrointest_disord/">gastrointest_disord</a>  <sub>OPT</sub><ul>
<li>Description: History of gastrointestinal tract disorders; can include multiple disorders. History of blood disorders; can include multiple disorders.  The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, gastrointestinal system disease (https://disease-ontology.org/?id=DOID:77).</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../geo_loc_name/">geo_loc_name</a>  <sub>OPT</sub><ul>
<li>Description: The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (v 1.512) (http://purl.bioontology.org/ontology/GAZ)</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: Germany;North Rhine-Westphalia;Eifel National Park None</li>
</ul>
</li>
<li><a href="../host_age/">host_age</a>  <sub>OPT</sub><ul>
<li>Description: Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees</li>
<li>range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 10 days None</li>
</ul>
</li>
<li><a href="../host_body_mass_index/">host_body_mass_index</a>  <sub>OPT</sub><ul>
<li>Description: Body mass index, calculated as weight/(height)squared</li>
<li>range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 22 kilogram per square meter None</li>
</ul>
</li>
<li><a href="../host_body_product/">host_body_product</a>  <sub>OPT</sub><ul>
<li>Description: Substance produced by the body, e.g. Stool, mucus, where the sample was obtained from. For foundational model of anatomy ontology (fma) or Uber-anatomy ontology (UBERON) terms, please see https://www.ebi.ac.uk/ols/ontologies/fma or https://www.ebi.ac.uk/ols/ontologies/uberon</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: Portion of mucus [fma66938] None</li>
</ul>
</li>
<li><a href="../host_body_site/">host_body_site</a>  <sub>OPT</sub><ul>
<li>Description: Name of body site where the sample was obtained from, such as a specific organ or tissue (tongue, lung etc...). For foundational model of anatomy ontology (fma) (v 4.11.0) or Uber-anatomy ontology (UBERON) (v releases/2014-06-15) terms, please see http://purl.bioontology.org/ontology/FMA or http://purl.bioontology.org/ontology/UBERON</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: gill [UBERON:0002535] None</li>
</ul>
</li>
<li><a href="../host_body_temp/">host_body_temp</a>  <sub>OPT</sub><ul>
<li>Description: Core body temperature of the host when sample was collected</li>
<li>range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 15 degree Celsius None</li>
</ul>
</li>
<li><a href="../host_diet/">host_diet</a>  <sub>OPT</sub><ul>
<li>Description: Type of diet depending on the host, for animals omnivore, herbivore etc., for humans high-fat, meditteranean etc.; can include multiple diet types</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: herbivore None</li>
</ul>
</li>
<li><a href="../host_disease_stat/">host_disease_stat</a>  <sub>OPT</sub><ul>
<li>Description: List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: rabies [DOID:11260] None</li>
</ul>
</li>
<li><a href="../host_family_relationship/">host_family_relationship</a>  <sub>OPT</sub><ul>
<li>Description: Relationships to other hosts in the same study; can include multiple relationships</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: offspring;Mussel25 None</li>
</ul>
</li>
<li><a href="../host_genotype/">host_genotype</a>  <sub>OPT</sub><ul>
<li>Description: Observed genotype</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: C57BL/6 None</li>
</ul>
</li>
<li><a href="../host_height/">host_height</a>  <sub>OPT</sub><ul>
<li>Description: The height of subject</li>
<li>range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 0.1 meter None</li>
</ul>
</li>
<li><a href="../host_last_meal/">host_last_meal</a>  <sub>OPT</sub><ul>
<li>Description: Content of last meal and time since feeding; can include multiple values</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: corn feed;P2H None</li>
</ul>
</li>
<li><a href="../host_occupation/">host_occupation</a>  <sub>OPT</sub><ul>
<li>Description: Most frequent job performed by subject</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: veterinary None</li>
</ul>
</li>
<li><a href="../host_phenotype/">host_phenotype</a>  <sub>OPT</sub><ul>
<li>Description: Phenotype of human or other host. For phenotypic quality ontology (pato) (v 2018-03-27) terms, please see http://purl.bioontology.org/ontology/pato. For Human Phenotype Ontology (HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: elongated [PATO:0001154] None</li>
</ul>
</li>
<li><a href="../host_pulse/">host_pulse</a>  <sub>OPT</sub><ul>
<li>Description: Resting pulse, measured as beats per minute</li>
<li>range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 65 beats per minute None</li>
</ul>
</li>
<li><a href="../host_sex/">host_sex</a>  <sub>OPT</sub><ul>
<li>Description: Physical sex of the host</li>
<li>range: </li>
<li>Example: female None</li>
</ul>
</li>
<li><a href="../host_subject_id/">host_subject_id</a>  <sub>OPT</sub><ul>
<li>Description: A unique identifier by which each subject can be referred to, de-identified, e.g. #131</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: MPI123 None</li>
</ul>
</li>
<li><a href="../host_symbiont/">host_symbiont</a>  <sub>OPT</sub><ul>
<li>Description: The taxonomic name of the organism(s) found living in mutualistic, commensalistic, or parasitic symbiosis with the specific host.</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: flukeworms None</li>
</ul>
</li>
<li><a href="../host_tot_mass/">host_tot_mass</a>  <sub>OPT</sub><ul>
<li>Description: Total mass of the host at collection, the unit depends on host</li>
<li>range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 2500 gram None</li>
</ul>
</li>
<li><a href="../ihmc_ethnicity/">ihmc_ethnicity</a>  <sub>OPT</sub><ul>
<li>Description: Ethnicity of the subject</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: caucasian None</li>
</ul>
</li>
<li><a href="../ihmc_medication_code/">ihmc_medication_code</a>  <sub>OPT</sub><ul>
<li>Description: Can include multiple medication codes</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: 810 None</li>
</ul>
</li>
<li><a href="../lat_lon/">lat_lon</a>  <sub>OPT</sub><ul>
<li>Description: The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: 50.586825 6.408977 None</li>
</ul>
</li>
<li><a href="../liver_disord/">liver_disord</a>  <sub>OPT</sub><ul>
<li>Description: History of liver disorders; can include multiple disorders. The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, liver disease (https://disease-ontology.org/?id=DOID:409).</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../medic_hist_perform/">medic_hist_perform</a>  <sub>OPT</sub><ul>
<li>Description: Whether full medical history was collected</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: TRUE None</li>
</ul>
</li>
<li><a href="../misc_param/">misc_param</a>  <sub>OPT</sub><ul>
<li>Description: Any other measurement performed or parameter collected, that is not listed here</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: Bicarbonate ion concentration;2075 micromole per kilogram None</li>
</ul>
</li>
<li><a href="../organism_count/">organism_count</a>  <sub>OPT</sub><ul>
<li>Description: Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)</li>
<li>range: </li>
<li>Example: total prokaryotes;3.5e7 cells per milliliter;qPCR None</li>
</ul>
</li>
<li><a href="../oxy_stat_samp/">oxy_stat_samp</a>  <sub>OPT</sub><ul>
<li>Description: Oxygenation status of sample</li>
<li>range: </li>
<li>Example: aerobic None</li>
</ul>
</li>
<li><a href="../perturbation/">perturbation</a>  <sub>OPT</sub><ul>
<li>Description: Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M None</li>
</ul>
</li>
<li><a href="../samp_salinity/">samp_salinity</a>  <sub>OPT</sub><ul>
<li>Description: Salinity is the total concentration of all dissolved salts in a liquid or solid (in the form of an extract obtained by centrifugation) sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater</li>
<li>range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 1 milligram per liter None</li>
</ul>
</li>
<li><a href="../samp_store_dur/">samp_store_dur</a>  <sub>OPT</sub><ul>
<li>Description: Duration for which the sample was stored</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: P1Y6M None</li>
</ul>
</li>
<li><a href="../samp_store_loc/">samp_store_loc</a>  <sub>OPT</sub><ul>
<li>Description: Location at which sample was stored, usually name of a specific freezer/room</li>
<li>range: <a href="../types/String/">String</a></li>
<li>Example: Freezer no:5 None</li>
</ul>
</li>
<li><a href="../samp_store_temp/">samp_store_temp</a>  <sub>OPT</sub><ul>
<li>Description: Temperature at which sample was stored, e.g. -80 degree Celsius</li>
<li>range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: -80 degree Celsius None</li>
</ul>
</li>
<li><a href="../samp_vol_we_dna_ext/">samp_vol_we_dna_ext</a>  <sub>OPT</sub><ul>
<li>Description: Volume (ml), weight (g) of processed sample, or surface area swabbed from sample for DNA extraction</li>
<li>range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 1500 milliliter None</li>
</ul>
</li>
<li><a href="../special_diet/">special_diet</a>  <sub>OPT</sub><ul>
<li>Description: Specification of special diet; can include multiple special diets</li>
<li>range: </li>
<li>Example: other:vegan None</li>
</ul>
</li>
<li><a href="../temp/">temp</a>  <sub>OPT</sub><ul>
<li>Description: Temperature of the sample at the time of sampling</li>
<li>range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 25 degree Celsius None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_adapters/">MIGS bacteria➞adapters</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_1">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_annot/">MIGS bacteria➞annot</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_2">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_assembly_name/">MIGS bacteria➞assembly_name</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_3">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_assembly_qual/">MIGS bacteria➞assembly_qual</a>  <sub>REQ</sub><ul>
<li>range: </li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_4">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_assembly_software/">MIGS bacteria➞assembly_software</a>  <sub>REQ</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_5">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_biotic_relationship/">MIGS bacteria➞biotic_relationship</a>  <sub>OPT</sub><ul>
<li>range: </li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_6">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_compl_score/">MIGS bacteria➞compl_score</a>  <sub>OPT</sub><ul>
<li>range: </li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_7">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_compl_software/">MIGS bacteria➞compl_software</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_8">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_encoded_traits/">MIGS bacteria➞encoded_traits</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_9">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_env_package/">MIGS bacteria➞env_package</a>  <sub>OPT</sub><ul>
<li>range: </li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_10">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_estimated_size/">MIGS bacteria➞estimated_size</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_11">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_experimental_factor/">MIGS bacteria➞experimental_factor</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_12">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_extrachrom_elements/">MIGS bacteria➞extrachrom_elements</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_13">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_feat_pred/">MIGS bacteria➞feat_pred</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_14">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_health_disease_stat/">MIGS bacteria➞health_disease_stat</a>  <sub>OPT</sub><ul>
<li>range: </li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_15">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_host_spec_range/">MIGS bacteria➞host_spec_range</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_16">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_investigation_type/">MIGS bacteria➞investigation_type</a>  <sub>REQ</sub><ul>
<li>range: </li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_17">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_isol_growth_condt/">MIGS bacteria➞isol_growth_condt</a>  <sub>REQ</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_18">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_lib_layout/">MIGS bacteria➞lib_layout</a>  <sub>OPT</sub><ul>
<li>range: </li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_19">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_lib_reads_seqd/">MIGS bacteria➞lib_reads_seqd</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_20">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_lib_screen/">MIGS bacteria➞lib_screen</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_21">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_lib_size/">MIGS bacteria➞lib_size</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_22">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_lib_vector/">MIGS bacteria➞lib_vector</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_23">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_nucl_acid_amp/">MIGS bacteria➞nucl_acid_amp</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_24">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_nucl_acid_ext/">MIGS bacteria➞nucl_acid_ext</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_25">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_num_replicons/">MIGS bacteria➞num_replicons</a>  <sub>REQ</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_26">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_number_contig/">MIGS bacteria➞number_contig</a>  <sub>REQ</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_27">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_pathogenicity/">MIGS bacteria➞pathogenicity</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_28">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_project_name/">MIGS bacteria➞project_name</a>  <sub>REQ</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_29">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_ref_biomaterial/">MIGS bacteria➞ref_biomaterial</a>  <sub>REQ</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_30">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_ref_db/">MIGS bacteria➞ref_db</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_31">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_rel_to_oxygen/">MIGS bacteria➞rel_to_oxygen</a>  <sub>OPT</sub><ul>
<li>range: </li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_32">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_samp_mat_process/">MIGS bacteria➞samp_mat_process</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_33">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_samp_size/">MIGS bacteria➞samp_size</a>  <sub>OPT</sub><ul>
<li>range: <a href="../QuantityValue/">QuantityValue</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_34">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_sample_collect_device/">MIGS bacteria➞sample_collect_device</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_35">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_sample_collect_method/">MIGS bacteria➞sample_collect_method</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_36">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_sample_name/">MIGS bacteria➞sample_name</a>  <sub>REQ</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_37">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_seq_meth/">MIGS bacteria➞seq_meth</a>  <sub>REQ</sub><ul>
<li>range: </li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_38">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_sim_search_meth/">MIGS bacteria➞sim_search_meth</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_39">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_sop/">MIGS bacteria➞sop</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_40">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_source_mat_id/">MIGS bacteria➞source_mat_id</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_41">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_specific_host/">MIGS bacteria➞specific_host</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_42">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_submitted_to_insdc/">MIGS bacteria➞submitted_to_insdc</a>  <sub>REQ</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_43">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_subspecf_gen_lin/">MIGS bacteria➞subspecf_gen_lin</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_44">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_tax_class/">MIGS bacteria➞tax_class</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_45">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_tax_ident/">MIGS bacteria➞tax_ident</a>  <sub>OPT</sub><ul>
<li>range: </li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_46">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_trophic_level/">MIGS bacteria➞trophic_level</a>  <sub>OPT</sub><ul>
<li>range: </li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_47">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_url/">MIGS bacteria➞url</a>  <sub>OPT</sub><ul>
<li>range: <a href="../types/String/">String</a></li>
</ul>
</li>
</ul>
              
            </div>
          </div>
          <footer>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="versions">
    <span class="rst-current-version" data-toggle="rst-current-version">
      
      
      
    </span>
</div>
    <script>var base_url = '..';</script>
    <script src="../js/theme.js" defer></script>
      <script src="../search/main.js" defer></script>
    <script defer>
        window.onload = function () {
            SphinxRtdTheme.Navigation.enable(true);
        };
    </script>

</body>
</html>
