<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  
  <link rel="canonical" href="https://cmungall.github.io/mixs-source/Core/">
  <link rel="shortcut icon" href="../img/favicon.ico">
  <title>Class: core - MIxS</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700" />

  <link rel="stylesheet" href="../css/theme.css" />
  <link rel="stylesheet" href="../css/theme_extra.css" />
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css" />
  
  <script>
    // Current page data
    var mkdocs_page_name = "Class: core";
    var mkdocs_page_input_path = "Core.md";
    var mkdocs_page_url = "/mixs-source/Core/";
  </script>
  
  <script src="../js/jquery-2.1.1.min.js" defer></script>
  <script src="../js/modernizr-2.8.3.min.js" defer></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
  <script>hljs.initHighlightingOnLoad();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> MIxS</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="..">Index</a>
                    </li>
                </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">MIxS</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
    
    <li>Class: core</li>
    <li class="wy-breadcrumbs-aside">
      
        <a href="https://github.com/cmungall/mixs-source/edit/master/docs/Core.md"
          class="icon icon-github"> Edit on GitHub</a>
      
    </li>
  </ul>
  
  <hr/>
</div>

          <div role="main">
            <div class="section">
              
                <h1 id="class-core">Class: core</h1>
<p>Core package. Do not use this directly, this is used to build other packages</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/Core">mixs.vocab:Core</a></p>
<p><a href="https://yuml.me/diagram/nofunky;dir:TB/class/[QuantityValue],[QuantityValue]&lt;samp_vol_we_dna_ext%200..1-++[Core&#124;submitted_to_insdc:string%20%3F;investigation_type:investigation_type_enum%20%3F;samp_name:string%20%3F;samp_taxon_id:string%20%3F;project_name:string%20%3F;experimental_factor:string%20%3F;lat_lon:string%20%3F;geo_loc_name:string%20%3F;collection_date:date%20%3F;env_broad_scale:string%20%3F;env_local_scale:string%20%3F;env_medium:string%20%3F;env_package:env_package_enum%20%3F;subspecf_gen_lin:string%20%3F;ploidy:string%20%3F;num_replicons:integer%20%3F;extrachrom_elements:integer%20%3F;estimated_size:string%20%3F;ref_biomaterial:string%20%3F;source_mat_id:string%20%3F;pathogenicity:string%20%3F;biotic_relationship:biotic_relationship_enum%20%3F;specific_host:string%20%3F;host_spec_range:integer%20%3F;health_disease_stat:health_disease_stat_enum%20%3F;host_disease_stat:string%20%3F;trophic_level:trophic_level_enum%20%3F;propagation:string%20%3F;encoded_traits:string%20%3F;rel_to_oxygen:rel_to_oxygen_enum%20%3F;isol_growth_condt:string%20%3F;samp_collec_device:string%20%3F;samp_collec_method:string%20%3F;samp_mat_process:string%20%3F;size_frac:string%20%3F;source_uvig:source_uvig_enum%20%3F;virus_enrich_appr:virus_enrich_appr_enum%20%3F;nucl_acid_ext:string%20%3F;nucl_acid_amp:string%20%3F;lib_size:integer%20%3F;lib_reads_seqd:integer%20%3F;lib_layout:lib_layout_enum%20%3F;lib_vector:string%20%3F;lib_screen:string%20%3F;target_gene:string%20%3F;target_subfragment:string%20%3F;pcr_primers:string%20%3F;mid:string%20%3F;adapters:string%20%3F;pcr_cond:string%20%3F;seq_meth:string%20%3F;seq_quality_check:string%20%3F;chimera_check:string%20%3F;tax_ident:tax_ident_enum%20%3F;assembly_qual:assembly_qual_enum%20%3F;assembly_name:string%20%3F;assembly_software:string%20%3F;annot:string%20%3F;number_contig:integer%20%3F;feat_pred:string%20%3F;ref_db:string%20%3F;sim_search_meth:string%20%3F;tax_class:string%20%3F;x_16s_recover:string%20%3F;trnas:integer%20%3F;trna_ext_software:string%20%3F;compl_score:compl_score_enum%20%3F;compl_software:string%20%3F;compl_appr:compl_appr_enum%20%3F;contam_score:string%20%3F;contam_screen_input:string%20%3F;contam_screen_param:contam_screen_param_enum%20%3F;decontam_software:decontam_software_enum%20%3F;sort_tech:sort_tech_enum%20%3F;single_cell_lysis_appr:single_cell_lysis_appr_enum%20%3F;single_cell_lysis_prot:string%20%3F;wga_amp_appr:string%20%3F;wga_amp_kit:string%20%3F;bin_param:bin_param_enum%20%3F;bin_software:string%20%3F;reassembly_bin:string%20%3F;mag_cov_software:mag_cov_software_enum%20%3F;vir_ident_software:string%20%3F;pred_genome_type:pred_genome_type_enum%20%3F;pred_genome_struc:pred_genome_struc_enum%20%3F;detec_type:string%20%3F;otu_class_appr:string%20%3F;otu_seq_comp_appr:string%20%3F;otu_db:string%20%3F;host_pred_appr:host_pred_appr_enum%20%3F;host_pred_est_acc:string%20%3F;associated_resource:string%20%3F;sop:string%20%3F],[QuantityValue]&lt;samp_size%200..1-++[Core],[QuantityValue]&lt;temp%200..1-++[Core],[QuantityValue]&lt;elev%200..1-++[Core],[QuantityValue]&lt;alt%200..1-++[Core],[QuantityValue]&lt;depth%200..1-++[Core]"><img alt="img" src="https://yuml.me/diagram/nofunky;dir:TB/class/[QuantityValue],[QuantityValue]&lt;samp_vol_we_dna_ext%200..1-++[Core&#124;submitted_to_insdc:string%20%3F;investigation_type:investigation_type_enum%20%3F;samp_name:string%20%3F;samp_taxon_id:string%20%3F;project_name:string%20%3F;experimental_factor:string%20%3F;lat_lon:string%20%3F;geo_loc_name:string%20%3F;collection_date:date%20%3F;env_broad_scale:string%20%3F;env_local_scale:string%20%3F;env_medium:string%20%3F;env_package:env_package_enum%20%3F;subspecf_gen_lin:string%20%3F;ploidy:string%20%3F;num_replicons:integer%20%3F;extrachrom_elements:integer%20%3F;estimated_size:string%20%3F;ref_biomaterial:string%20%3F;source_mat_id:string%20%3F;pathogenicity:string%20%3F;biotic_relationship:biotic_relationship_enum%20%3F;specific_host:string%20%3F;host_spec_range:integer%20%3F;health_disease_stat:health_disease_stat_enum%20%3F;host_disease_stat:string%20%3F;trophic_level:trophic_level_enum%20%3F;propagation:string%20%3F;encoded_traits:string%20%3F;rel_to_oxygen:rel_to_oxygen_enum%20%3F;isol_growth_condt:string%20%3F;samp_collec_device:string%20%3F;samp_collec_method:string%20%3F;samp_mat_process:string%20%3F;size_frac:string%20%3F;source_uvig:source_uvig_enum%20%3F;virus_enrich_appr:virus_enrich_appr_enum%20%3F;nucl_acid_ext:string%20%3F;nucl_acid_amp:string%20%3F;lib_size:integer%20%3F;lib_reads_seqd:integer%20%3F;lib_layout:lib_layout_enum%20%3F;lib_vector:string%20%3F;lib_screen:string%20%3F;target_gene:string%20%3F;target_subfragment:string%20%3F;pcr_primers:string%20%3F;mid:string%20%3F;adapters:string%20%3F;pcr_cond:string%20%3F;seq_meth:string%20%3F;seq_quality_check:string%20%3F;chimera_check:string%20%3F;tax_ident:tax_ident_enum%20%3F;assembly_qual:assembly_qual_enum%20%3F;assembly_name:string%20%3F;assembly_software:string%20%3F;annot:string%20%3F;number_contig:integer%20%3F;feat_pred:string%20%3F;ref_db:string%20%3F;sim_search_meth:string%20%3F;tax_class:string%20%3F;x_16s_recover:string%20%3F;trnas:integer%20%3F;trna_ext_software:string%20%3F;compl_score:compl_score_enum%20%3F;compl_software:string%20%3F;compl_appr:compl_appr_enum%20%3F;contam_score:string%20%3F;contam_screen_input:string%20%3F;contam_screen_param:contam_screen_param_enum%20%3F;decontam_software:decontam_software_enum%20%3F;sort_tech:sort_tech_enum%20%3F;single_cell_lysis_appr:single_cell_lysis_appr_enum%20%3F;single_cell_lysis_prot:string%20%3F;wga_amp_appr:string%20%3F;wga_amp_kit:string%20%3F;bin_param:bin_param_enum%20%3F;bin_software:string%20%3F;reassembly_bin:string%20%3F;mag_cov_software:mag_cov_software_enum%20%3F;vir_ident_software:string%20%3F;pred_genome_type:pred_genome_type_enum%20%3F;pred_genome_struc:pred_genome_struc_enum%20%3F;detec_type:string%20%3F;otu_class_appr:string%20%3F;otu_seq_comp_appr:string%20%3F;otu_db:string%20%3F;host_pred_appr:host_pred_appr_enum%20%3F;host_pred_est_acc:string%20%3F;associated_resource:string%20%3F;sop:string%20%3F],[QuantityValue]&lt;samp_size%200..1-++[Core],[QuantityValue]&lt;temp%200..1-++[Core],[QuantityValue]&lt;elev%200..1-++[Core],[QuantityValue]&lt;alt%200..1-++[Core],[QuantityValue]&lt;depth%200..1-++[Core]" /></a></p>
<h2 id="attributes">Attributes</h2>
<h3 id="own">Own</h3>
<ul>
<li><a href="../submitted_to_insdc/">submitted_to_insdc</a>  <sub>0..1</sub><ul>
<li>Description: Depending on the study (large-scale e.g. done with next generation sequencing technology, or small-scale) sequences have to be submitted to SRA (Sequence Read Archive), DRA (DDBJ Read Archive) or via the classical Webin/Sequin systems to Genbank, ENA and DDBJ. Although this field is mandatory, it is meant as a self-test field, therefore it is not necessary to include this field in contextual data submitted to databases</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: yes None</li>
</ul>
</li>
<li><a href="../investigation_type/">investigation_type</a>  <sub>0..1</sub><ul>
<li>Description: Nucleic Acid Sequence Report is the root element of all MIGS/MIMS compliant reports as standardized by Genomic Standards Consortium. This field is either eukaryote,bacteria,virus,plasmid,organelle, metagenome,mimarks-survey, mimarks-specimen, metatranscriptome, single amplified genome, metagenome-assembled genome, or uncultivated viral genome</li>
<li>Range: <a href="../investigation_type_enum/">investigation_type_enum</a></li>
<li>Example: metagenome None</li>
</ul>
</li>
<li><a href="../samp_name/">samp_name</a>  <sub>0..1</sub><ul>
<li>Description: A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: ISDsoil1 None</li>
</ul>
</li>
<li><a href="../samp_taxon_id/">samp_taxon_id</a>  <sub>0..1</sub><ul>
<li>Description: NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Gut Metagenome [NCBI:txid749906] None</li>
</ul>
</li>
<li><a href="../project_name/">project_name</a>  <sub>0..1</sub><ul>
<li>Description: Name of the project within which the sequencing was organized</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Forest soil metagenome None</li>
</ul>
</li>
<li><a href="../experimental_factor/">experimental_factor</a>  <sub>0..1</sub><ul>
<li>Description: Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: time series design [EFO:EFO_0001779] None</li>
</ul>
</li>
<li><a href="../lat_lon/">lat_lon</a>  <sub>0..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 50.586825 6.408977 None</li>
</ul>
</li>
<li><a href="../depth/">depth</a>  <sub>0..1</sub><ul>
<li>Description: The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 10 meter None</li>
</ul>
</li>
<li><a href="../alt/">alt</a>  <sub>0..1</sub><ul>
<li>Description: Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../elev/">elev</a>  <sub>0..1</sub><ul>
<li>Description: Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../temp/">temp</a>  <sub>0..1</sub><ul>
<li>Description: Temperature of the sample at the time of sampling.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 25 degree Celsius None</li>
</ul>
</li>
<li><a href="../geo_loc_name/">geo_loc_name</a>  <sub>0..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: USA: Maryland, Bethesda None</li>
</ul>
</li>
<li><a href="../collection_date/">collection_date</a>  <sub>0..1</sub><ul>
<li>Description: The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</li>
<li>Range: <a href="../types/Date/">Date</a></li>
<li>Example: 2018-05-11T10:00:00+01:00; 2018-05-11 None</li>
</ul>
</li>
<li><a href="../env_broad_scale/">env_broad_scale</a>  <sub>0..1</sub><ul>
<li>Description: Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: oceanic epipelagic zone biome [ENVO:01000033] for annotating a water sample from the photic zone in middle of the Atlantic Ocean None</li>
</ul>
</li>
<li><a href="../env_local_scale/">env_local_scale</a>  <sub>0..1</sub><ul>
<li>Description: Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: litter layer [ENVO:01000338]; Annotating a pooled sample taken from various vegetation layers in a forest consider: canopy [ENVO:00000047]|herb and fern layer [ENVO:01000337]|litter layer [ENVO:01000338]|understory [01000335]|shrub layer [ENVO:01000336]. None</li>
</ul>
</li>
<li><a href="../env_medium/">env_medium</a>  <sub>0..1</sub><ul>
<li>Description: Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: soil [ENVO:00001998]; Annotating a fish swimming in the upper 100 m of the Atlantic Ocean, consider: ocean water [ENVO:00002151]. Example: Annotating a duck on a pond consider: pond water [ENVO:00002228]|air [ENVO_00002005] None</li>
</ul>
</li>
<li><a href="../env_package/">env_package</a>  <sub>0..1</sub><ul>
<li>Description: MIxS extension for reporting of measurements and observations obtained from one or more of the environments where the sample was obtained. All environmental packages listed here are further defined in separate subtables. By giving the name of the environmental package, a selection of fields can be made from the subtables and can be reported</li>
<li>Range: <a href="../env_package_enum/">env_package_enum</a></li>
<li>Example: soil None</li>
</ul>
</li>
<li><a href="../subspecf_gen_lin/">subspecf_gen_lin</a>  <sub>0..1</sub><ul>
<li>Description: Information about the genetic distinctness of the sequenced organism below the subspecies level, e.g., serovar, serotype, biotype, ecotype, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: serovar:Newport None</li>
</ul>
</li>
<li><a href="../ploidy/">ploidy</a>  <sub>0..1</sub><ul>
<li>Description: The ploidy level of the genome (e.g. allopolyploid, haploid, diploid, triploid, tetraploid). It has implications for the downstream study of duplicated gene and regions of the genomes (and perhaps for difficulties in assembly). For terms, please select terms listed under class ploidy (PATO:001374) of Phenotypic Quality Ontology (PATO), and for a browser of PATO (v 2018-03-27) please refer to http://purl.bioontology.org/ontology/PATO</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: allopolyploidy [PATO:0001379] None</li>
</ul>
</li>
<li><a href="../num_replicons/">num_replicons</a>  <sub>0..1</sub><ul>
<li>Description: Reports the number of replicons in a nuclear genome of eukaryotes, in the genome of a bacterium or archaea or the number of segments in a segmented virus. Always applied to the haploid chromosome count of a eukaryote</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 2 None</li>
</ul>
</li>
<li><a href="../extrachrom_elements/">extrachrom_elements</a>  <sub>0..1</sub><ul>
<li>Description: Do plasmids exist of significant phenotypic consequence (e.g. ones that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids (borrelia has 15+ plasmids)</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 5 None</li>
</ul>
</li>
<li><a href="../estimated_size/">estimated_size</a>  <sub>0..1</sub><ul>
<li>Description: The estimated size of the genome prior to sequencing. Of particular importance in the sequencing of (eukaryotic) genome which could remain in draft form for a long or unspecified period.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 300000 bp None</li>
</ul>
</li>
<li><a href="../ref_biomaterial/">ref_biomaterial</a>  <sub>0..1</sub><ul>
<li>Description: Primary publication if isolated before genome publication; otherwise, primary genome report.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: doi:10.1016/j.syapm.2018.01.009 None</li>
</ul>
</li>
<li><a href="../source_mat_id/">source_mat_id</a>  <sub>0..1</sub><ul>
<li>Description: A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: MPI012345 None</li>
</ul>
</li>
<li><a href="../pathogenicity/">pathogenicity</a>  <sub>0..1</sub><ul>
<li>Description: To what is the entity pathogenic</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: human, animal, plant, fungi, bacteria None</li>
</ul>
</li>
<li><a href="../biotic_relationship/">biotic_relationship</a>  <sub>0..1</sub><ul>
<li>Description: Description of relationship(s) between the subject organism and other organism(s) it is associated with. E.g., parasite on species X; mutualist with species Y. The target organism is the subject of the relationship, and the other organism(s) is the object</li>
<li>Range: <a href="../biotic_relationship_enum/">biotic_relationship_enum</a></li>
<li>Example: free living None</li>
</ul>
</li>
<li><a href="../specific_host/">specific_host</a>  <sub>0..1</sub><ul>
<li>Description: Report the host's taxonomic name and/or NCBI taxonomy ID.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Homo sapiens and/or 9606 None</li>
</ul>
</li>
<li><a href="../host_spec_range/">host_spec_range</a>  <sub>0..1</sub><ul>
<li>Description: The range and diversity of host species that an organism is capable of infecting, defined by NCBI taxonomy identifier.</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 9606 None</li>
</ul>
</li>
<li><a href="../health_disease_stat/">health_disease_stat</a>  <sub>0..1</sub><ul>
<li>Description: Health or disease status of specific host at time of collection</li>
<li>Range: <a href="../health_disease_stat_enum/">health_disease_stat_enum</a></li>
<li>Example: dead None</li>
</ul>
</li>
<li><a href="../host_disease_stat/">host_disease_stat</a>  <sub>0..1</sub><ul>
<li>Description: List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: rabies [DOID:11260] None</li>
</ul>
</li>
<li><a href="../trophic_level/">trophic_level</a>  <sub>0..1</sub><ul>
<li>Description: Trophic levels are the feeding position in a food chain. Microbes can be a range of producers (e.g. chemolithotroph)</li>
<li>Range: <a href="../trophic_level_enum/">trophic_level_enum</a></li>
<li>Example: heterotroph None</li>
</ul>
</li>
<li><a href="../propagation/">propagation</a>  <sub>0..1</sub><ul>
<li>Description: The type of reproduction from the parent stock. Values for this field is specific to different taxa. For phage or virus: lytic/lysogenic/temperate/obligately lytic. For plasmids: incompatibility group. For eukaryotes: sexual/asexual.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: lytic None</li>
</ul>
</li>
<li><a href="../encoded_traits/">encoded_traits</a>  <sub>0..1</sub><ul>
<li>Description: Should include key traits like antibiotic resistance or xenobiotic degradation phenotypes for plasmids, converting genes for phage</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: beta-lactamase class A None</li>
</ul>
</li>
<li><a href="../rel_to_oxygen/">rel_to_oxygen</a>  <sub>0..1</sub><ul>
<li>Description: Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments</li>
<li>Range: <a href="../rel_to_oxygen_enum/">rel_to_oxygen_enum</a></li>
<li>Example: aerobe None</li>
</ul>
</li>
<li><a href="../isol_growth_condt/">isol_growth_condt</a>  <sub>0..1</sub><ul>
<li>Description: Publication reference in the form of pubmed ID (pmid), digital object identifier (doi) or url for isolation and growth condition specifications of the organism/material</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: doi: 10.1016/j.syapm.2018.01.009 None</li>
</ul>
</li>
<li><a href="../samp_collec_device/">samp_collec_device</a>  <sub>0..1</sub><ul>
<li>Description: The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713] None</li>
</ul>
</li>
<li><a href="../samp_collec_method/">samp_collec_method</a>  <sub>0..1</sub><ul>
<li>Description: The method employed for collecting the sample.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: swabbing None</li>
</ul>
</li>
<li><a href="../samp_mat_process/">samp_mat_process</a>  <sub>0..1</sub><ul>
<li>Description: A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: filtering of seawater, storing samples in ethanol None</li>
</ul>
</li>
<li><a href="../size_frac/">size_frac</a>  <sub>0..1</sub><ul>
<li>Description: Filtering pore size used in sample preparation</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 0-0.22 micrometer None</li>
</ul>
</li>
<li><a href="../samp_size/">samp_size</a>  <sub>0..1</sub><ul>
<li>Description: The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 5 liter None</li>
</ul>
</li>
<li><a href="../samp_vol_we_dna_ext/">samp_vol_we_dna_ext</a>  <sub>0..1</sub><ul>
<li>Description: Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 1500 milliliter None</li>
</ul>
</li>
<li><a href="../source_uvig/">source_uvig</a>  <sub>0..1</sub><ul>
<li>Description: Type of dataset from which the UViG was obtained</li>
<li>Range: <a href="../source_uvig_enum/">source_uvig_enum</a></li>
<li>Example: viral fraction metagenome (virome) None</li>
</ul>
</li>
<li><a href="../virus_enrich_appr/">virus_enrich_appr</a>  <sub>0..1</sub><ul>
<li>Description: List of approaches used to enrich the sample for viruses, if any</li>
<li>Range: <a href="../virus_enrich_appr_enum/">virus_enrich_appr_enum</a></li>
<li>Example: filtration + FeCl Precipitation + ultracentrifugation + DNAse None</li>
</ul>
</li>
<li><a href="../nucl_acid_ext/">nucl_acid_ext</a>  <sub>0..1</sub><ul>
<li>Description: A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf None</li>
</ul>
</li>
<li><a href="../nucl_acid_amp/">nucl_acid_amp</a>  <sub>0..1</sub><ul>
<li>Description: A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://phylogenomics.me/protocols/16s-pcr-protocol/ None</li>
</ul>
</li>
<li><a href="../lib_size/">lib_size</a>  <sub>0..1</sub><ul>
<li>Description: Total number of clones in the library prepared for the project</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 50 None</li>
</ul>
</li>
<li><a href="../lib_reads_seqd/">lib_reads_seqd</a>  <sub>0..1</sub><ul>
<li>Description: Total number of clones sequenced from the library</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 20 None</li>
</ul>
</li>
<li><a href="../lib_layout/">lib_layout</a>  <sub>0..1</sub><ul>
<li>Description: Specify whether to expect single, paired, or other configuration of reads</li>
<li>Range: <a href="../lib_layout_enum/">lib_layout_enum</a></li>
<li>Example: paired None</li>
</ul>
</li>
<li><a href="../lib_vector/">lib_vector</a>  <sub>0..1</sub><ul>
<li>Description: Cloning vector type(s) used in construction of libraries</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Bacteriophage P1 None</li>
</ul>
</li>
<li><a href="../lib_screen/">lib_screen</a>  <sub>0..1</sub><ul>
<li>Description: Specific enrichment or screening methods applied before and/or after creating libraries</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: enriched, screened, normalized None</li>
</ul>
</li>
<li><a href="../target_gene/">target_gene</a>  <sub>0..1</sub><ul>
<li>Description: Targeted gene or locus name for marker gene studies</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 16S rRNA, 18S rRNA, nif, amoA, rpo None</li>
</ul>
</li>
<li><a href="../target_subfragment/">target_subfragment</a>  <sub>0..1</sub><ul>
<li>Description: Name of subfragment of a gene or locus. Important to e.g. identify special regions on marker genes like V6 on 16S rRNA</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: V6, V9, ITS None</li>
</ul>
</li>
<li><a href="../pcr_primers/">pcr_primers</a>  <sub>0..1</sub><ul>
<li>Description: PCR primers that were used to amplify the sequence of the targeted gene, locus or subfragment. This field should contain all the primers used for a single PCR reaction if multiple forward or reverse primers are present in a single PCR reaction. The primer sequence should be reported in uppercase letters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: FWD:GTGCCAGCMGCCGCGGTAA;REV:GGACTACHVGGGTWTCTAAT None</li>
</ul>
</li>
<li><a href="../mid/">mid</a>  <sub>0..1</sub><ul>
<li>Description: Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: GTGAATAT None</li>
</ul>
</li>
<li><a href="../adapters/">adapters</a>  <sub>0..1</sub><ul>
<li>Description: Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT None</li>
</ul>
</li>
<li><a href="../pcr_cond/">pcr_cond</a>  <sub>0..1</sub><ul>
<li>Description: Description of reaction conditions and components of PCR in the form of 'initial denaturation:94degC_1.5min; annealing=...'</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: initial denaturation:94_3;annealing:50_1;elongation:72_1.5;final elongation:72_10;35 None</li>
</ul>
</li>
<li><a href="../seq_meth/">seq_meth</a>  <sub>0..1</sub><ul>
<li>Description: Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 454 Genome Sequencer FLX [OBI:0000702] None</li>
</ul>
</li>
<li><a href="../seq_quality_check/">seq_quality_check</a>  <sub>0..1</sub><ul>
<li>Description: Indicate if the sequence has been called by automatic systems (none) or undergone a manual editing procedure (e.g. by inspecting the raw data or chromatograms). Applied only for sequences that are not submitted to SRA,ENA or DRA</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: none None</li>
</ul>
</li>
<li><a href="../chimera_check/">chimera_check</a>  <sub>0..1</sub><ul>
<li>Description: Tool(s) used for chimera checking, including version number and parameters, to discover and remove chimeric sequences. A chimeric sequence is comprised of two or more phylogenetically distinct parent sequences.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: uchime;v4.1;default parameters None</li>
</ul>
</li>
<li><a href="../tax_ident/">tax_ident</a>  <sub>0..1</sub><ul>
<li>Description: The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</li>
<li>Range: <a href="../tax_ident_enum/">tax_ident_enum</a></li>
<li>Example: other: rpoB gene None</li>
</ul>
</li>
<li><a href="../assembly_qual/">assembly_qual</a>  <sub>0..1</sub><ul>
<li>Description: The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for which no genome size could be estimated</li>
<li>Range: <a href="../assembly_qual_enum/">assembly_qual_enum</a></li>
<li>Example: High-quality draft genome None</li>
</ul>
</li>
<li><a href="../assembly_name/">assembly_name</a>  <sub>0..1</sub><ul>
<li>Description: Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: HuRef, JCVI_ISG_i3_1.0 None</li>
</ul>
</li>
<li><a href="../assembly_software/">assembly_software</a>  <sub>0..1</sub><ul>
<li>Description: Tool(s) used for assembly, including version number and parameters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise None</li>
</ul>
</li>
<li><a href="../annot/">annot</a>  <sub>0..1</sub><ul>
<li>Description: Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: prokka None</li>
</ul>
</li>
<li><a href="../number_contig/">number_contig</a>  <sub>0..1</sub><ul>
<li>Description: Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 40 None</li>
</ul>
</li>
<li><a href="../feat_pred/">feat_pred</a>  <sub>0..1</sub><ul>
<li>Description: Method used to predict UViGs features such as ORFs, integration site, etc.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Prodigal;2.6.3;default parameters None</li>
</ul>
</li>
<li><a href="../ref_db/">ref_db</a>  <sub>0..1</sub><ul>
<li>Description: List of database(s) used for ORF annotation, along with version number and reference to website or publication</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: pVOGs;5;http://dmk-brain.ecn.uiowa.edu/pVOGs/ Grazziotin et al. 2017 doi:10.1093/nar/gkw975 None</li>
</ul>
</li>
<li><a href="../sim_search_meth/">sim_search_meth</a>  <sub>0..1</sub><ul>
<li>Description: Tool used to compare ORFs with database, along with version and cutoffs used</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: HMMER3;3.1b2;hmmsearch, cutoff of 50 on score None</li>
</ul>
</li>
<li><a href="../tax_class/">tax_class</a>  <sub>0..1</sub><ul>
<li>Description: Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: vConTACT vContact2 (references from NCBI RefSeq v83, genus rank classification, default parameters) None</li>
</ul>
</li>
<li><a href="../x_16s_recover/">x_16s_recover</a>  <sub>0..1</sub><ul>
<li>Description: Can a 16S gene be recovered from the submitted SAG or MAG?</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: yes None</li>
</ul>
</li>
<li><a href="../trnas/">trnas</a>  <sub>0..1</sub><ul>
<li>Description: The total number of tRNAs identified from the SAG or MAG</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 18 None</li>
</ul>
</li>
<li><a href="../trna_ext_software/">trna_ext_software</a>  <sub>0..1</sub><ul>
<li>Description: Tools used for tRNA identification</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: infernal;v2;default parameters None</li>
</ul>
</li>
<li><a href="../compl_score/">compl_score</a>  <sub>0..1</sub><ul>
<li>Description: Completeness score is typically based on either the fraction of markers found as compared to a database or the percent of a genome found as compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated completeness scores</li>
<li>Range: <a href="../compl_score_enum/">compl_score_enum</a></li>
<li>Example: med;60% None</li>
</ul>
</li>
<li><a href="../compl_software/">compl_software</a>  <sub>0..1</sub><ul>
<li>Description: Tools used for completion estimate, i.e. checkm, anvi'o, busco</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: checkm None</li>
</ul>
</li>
<li><a href="../compl_appr/">compl_appr</a>  <sub>0..1</sub><ul>
<li>Description: The approach used to determine the completeness of a given genomic assembly, which would typically make use of a set of conserved marker genes or a closely related reference genome. For UViG completeness, include reference genome or group used, and contig feature suggesting a complete genome</li>
<li>Range: <a href="../compl_appr_enum/">compl_appr_enum</a></li>
<li>Example: other: UViG length compared to the average length of reference genomes from the P22virus genus (NCBI RefSeq v83) None</li>
</ul>
</li>
<li><a href="../contam_score/">contam_score</a>  <sub>0..1</sub><ul>
<li>Description: The contamination score is based on the fraction of single-copy genes that are observed more than once in a query genome. The following scores are acceptable for; High Quality Draft: &lt; 5%, Medium Quality Draft: &lt; 10%, Low Quality Draft: &lt; 10%. Contamination must be below 5% for a SAG or MAG to be deposited into any of the public databases</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 1% None</li>
</ul>
</li>
<li><a href="../contam_screen_input/">contam_screen_input</a>  <sub>0..1</sub><ul>
<li>Description: The type of sequence data used as input</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: contigs None</li>
</ul>
</li>
<li><a href="../contam_screen_param/">contam_screen_param</a>  <sub>0..1</sub><ul>
<li>Description: Specific parameters used in the decontamination sofware, such as reference database, coverage, and kmers. Combinations of these parameters may also be used, i.e. kmer and coverage, or reference database and kmer</li>
<li>Range: <a href="../contam_screen_param_enum/">contam_screen_param_enum</a></li>
<li>Example: kmer None</li>
</ul>
</li>
<li><a href="../decontam_software/">decontam_software</a>  <sub>0..1</sub><ul>
<li>Description: Tool(s) used in contamination screening</li>
<li>Range: <a href="../decontam_software_enum/">decontam_software_enum</a></li>
<li>Example: anvi'o None</li>
</ul>
</li>
<li><a href="../sort_tech/">sort_tech</a>  <sub>0..1</sub><ul>
<li>Description: Method used to sort/isolate cells or particles of interest</li>
<li>Range: <a href="../sort_tech_enum/">sort_tech_enum</a></li>
<li>Example: optical manipulation None</li>
</ul>
</li>
<li><a href="../single_cell_lysis_appr/">single_cell_lysis_appr</a>  <sub>0..1</sub><ul>
<li>Description: Method used to free DNA from interior of the cell(s) or particle(s)</li>
<li>Range: <a href="../single_cell_lysis_appr_enum/">single_cell_lysis_appr_enum</a></li>
<li>Example: enzymatic None</li>
</ul>
</li>
<li><a href="../single_cell_lysis_prot/">single_cell_lysis_prot</a>  <sub>0..1</sub><ul>
<li>Description: Name of the kit or standard protocol used for cell(s) or particle(s) lysis</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: ambion single cell lysis kit None</li>
</ul>
</li>
<li><a href="../wga_amp_appr/">wga_amp_appr</a>  <sub>0..1</sub><ul>
<li>Description: Method used to amplify genomic DNA in preparation for sequencing</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: mda based None</li>
</ul>
</li>
<li><a href="../wga_amp_kit/">wga_amp_kit</a>  <sub>0..1</sub><ul>
<li>Description: Kit used to amplify genomic DNA in preparation for sequencing</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: qiagen repli-g None</li>
</ul>
</li>
<li><a href="../bin_param/">bin_param</a>  <sub>0..1</sub><ul>
<li>Description: The parameters that have been applied during the extraction of genomes from metagenomic datasets</li>
<li>Range: <a href="../bin_param_enum/">bin_param_enum</a></li>
<li>Example: coverage and kmer None</li>
</ul>
</li>
<li><a href="../bin_software/">bin_software</a>  <sub>0..1</sub><ul>
<li>Description: Tool(s) used for the extraction of genomes from metagenomic datasets, where possible include a product ID (PID) of the tool(s) used.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: MetaCluster-TA (RRID:SCR_004599), MaxBin (biotools:maxbin) None</li>
</ul>
</li>
<li><a href="../reassembly_bin/">reassembly_bin</a>  <sub>0..1</sub><ul>
<li>Description: Has an assembly been performed on a genome bin extracted from a metagenomic assembly?</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: no None</li>
</ul>
</li>
<li><a href="../mag_cov_software/">mag_cov_software</a>  <sub>0..1</sub><ul>
<li>Description: Tool(s) used to determine the genome coverage if coverage is used as a binning parameter in the extraction of genomes from metagenomic datasets</li>
<li>Range: <a href="../mag_cov_software_enum/">mag_cov_software_enum</a></li>
<li>Example: bbmap None</li>
</ul>
</li>
<li><a href="../vir_ident_software/">vir_ident_software</a>  <sub>0..1</sub><ul>
<li>Description: Tool(s) used for the identification of UViG as a viral genome, software or protocol name including version number, parameters, and cutoffs used</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: VirSorter; 1.0.4; Virome database, category 2 None</li>
</ul>
</li>
<li><a href="../pred_genome_type/">pred_genome_type</a>  <sub>0..1</sub><ul>
<li>Description: Type of genome predicted for the UViG</li>
<li>Range: <a href="../pred_genome_type_enum/">pred_genome_type_enum</a></li>
<li>Example: dsDNA None</li>
</ul>
</li>
<li><a href="../pred_genome_struc/">pred_genome_struc</a>  <sub>0..1</sub><ul>
<li>Description: Expected structure of the viral genome</li>
<li>Range: <a href="../pred_genome_struc_enum/">pred_genome_struc_enum</a></li>
<li>Example: non-segmented None</li>
</ul>
</li>
<li><a href="../detec_type/">detec_type</a>  <sub>0..1</sub><ul>
<li>Description: Type of UViG detection</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: independent sequence (UViG) None</li>
</ul>
</li>
<li><a href="../otu_class_appr/">otu_class_appr</a>  <sub>0..1</sub><ul>
<li>Description: Cutoffs and approach used when clustering “species-level” OTUs. Note that results from standard 95% ANI / 85% AF clustering should be provided alongside OTUS defined from another set of thresholds, even if the latter are the ones primarily used during the analysis</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 95% ANI;85% AF; greedy incremental clustering None</li>
</ul>
</li>
<li><a href="../otu_seq_comp_appr/">otu_seq_comp_appr</a>  <sub>0..1</sub><ul>
<li>Description: Tool and thresholds used to compare sequences when computing "species-level" OTUs</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: blastn;2.6.0+;e-value cutoff: 0.001 None</li>
</ul>
</li>
<li><a href="../otu_db/">otu_db</a>  <sub>0..1</sub><ul>
<li>Description: Reference database (i.e. sequences not generated as part of the current study) used to cluster new genomes in "species-level" OTUs, if any</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: NCBI Viral RefSeq;83 None</li>
</ul>
</li>
<li><a href="../host_pred_appr/">host_pred_appr</a>  <sub>0..1</sub><ul>
<li>Description: Tool or approach used for host prediction</li>
<li>Range: <a href="../host_pred_appr_enum/">host_pred_appr_enum</a></li>
<li>Example: CRISPR spacer match None</li>
</ul>
</li>
<li><a href="../host_pred_est_acc/">host_pred_est_acc</a>  <sub>0..1</sub><ul>
<li>Description: For each tool or approach used for host prediction, estimated false discovery rates should be included, either computed de novo or from the literature</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: CRISPR spacer match: 0 or 1 mismatches, estimated 8% FDR at the host genus rank (Edwards et al. 2016 doi:10.1093/femsre/fuv048) None</li>
</ul>
</li>
<li><a href="../associated_resource/">associated resource</a>  <sub>0..1</sub><ul>
<li>Description: A related resource that is referenced, cited, or otherwise associated to the sequence.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://www.earthmicrobiome.org/ None</li>
</ul>
</li>
<li><a href="../sop/">sop</a>  <sub>0..1</sub><ul>
<li>Description: Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/ None</li>
</ul>
</li>
</ul>
              
            </div>
          </div>
          <footer>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
        <span>
          <a href="https://github.com/cmungall/mixs-source/" class="fa fa-github" style="color: #fcfcfc"> GitHub</a>
        </span>
    
    
    
  </span>
</div>
    <script>var base_url = '..';</script>
    <script src="../js/theme_extra.js" defer></script>
    <script src="../js/theme.js" defer></script>
      <script src="../search/main.js" defer></script>
    <script defer>
        window.onload = function () {
            SphinxRtdTheme.Navigation.enable(true);
        };
    </script>

</body>
</html>
